Docs/7_Security.md

```diff
@@ -1,5 +1,151 @@
 # Security
 
+Alchemy provides built in support for Bcrypt hashing and automatic authentication via Rune & `Middleware`.
+
+## Bcrypt
+
+A standard practice is to never store plain text passwords in your database. Bcrypt is a password hasing function that creats a one way hash of a plaintext password. It's a expensive CPU-wise, so it will help protect against your hashed passwords from being cracked through brute forced.
```

Comment:
Typo "password hashing function"

Reply:
And another "that creates"

Reply:
Verbiage tweak "it's an expensive process CPU-wise"

Reply:
"Help protect your passwords from being easily cracked through brute forcing" might be a better phrasing

Reply:
Lol I clearly wrote this at 1:30am